modules/go-add-infra-nodes.adoc

```diff
@@ -20,7 +34,7 @@ $ oc label node <node-name> node-role.kubernetes.io/infra=
 ----
 $ oc edit gitopsservice -n openshift-gitops
 ----
-. In the `GitOpsService` CR file, add `runOnInfra` toggle to the `spec` section and set it to `true`:
+. In the `GitOpsService` CR file, add `runOnInfra` toggle to the `spec` section and set it to `true`. This toggle will move the workloads in `openshift-gitops` namespace to the infrastructure nodes:
```

Comment:
```suggestion
. In the `GitOpsService` CR file, add `runOnInfra` field to the `spec` section and set it to `true`. This field moves the workloads in `openshift-gitops` namespace to the infrastructure nodes:
```
- "Toggle" is non-standard terminology. Please replace it with established terminology. 
- Avoid using future tense, "will."